### PR TITLE
Switches from ascii folding to icu folding.

### LINKED ISSUE #116: searching Sinopia is not finding some characters with diacritics
earlier in https://github.com/LD4P/sinopia_indexing_pipeline/issues/53 we improved the Sinopia search so that if you search for a simple character, it finds matches of that character with and without diacritical marks (example: search for `orbita` and results include `Órbita`

however there is at least one example where it looks like Sinopia things a character plus diacritic is actually a character followed by another character:
https://trellis.stage.sinopia.io/https://trellis.stage.sinopia.io/repository/chicago/a75ad20f-7107-48aa-8165-204cec78184a

the title is `Voĭna i mir`
user expects to finds this by searching for `voina` but `voina` gives 0 results

however searching for `voi*` with wildcard finds it, as if `ĭ` were 'i' followed by another character

`voina` should find `Voĭna`

I don't know what other diacritical marks this is happening with

(also, case is not an issue, we have case insensitive search and it's working as expected)

### PR DESCRIPTION
**Pull request by justinlittman**
closes #116

### CODE REVIEW

File: src/Indexer.js [hunk 1]
```diff
@@ -80,14 +80,14 @@ export default class Indexer {
     })
   }
 
-  // https://www.elastic.co/guide/en/elasticsearch/reference/current/analysis-asciifolding-tokenfilter.html
+  // hhttps://www.elastic.co/guide/en/elasticsearch/plugins/current/analysis-icu-folding.html
```
Comment (jcoyne) [inline src/Indexer.js]: Typo

**Comment by justinlittman** [comment]
No, I didn't get test to fail. Chalking it up to a difference between AWS and ES.

File: Dockerfile-es [hunk 2]
```diff
@@ -0,0 +1,3 @@
+FROM docker.elastic.co/elasticsearch/elasticsearch:6.4.3
```
Comment (mjgiarlo) [inline Dockerfile-es]: Can you refresh my memory: is this the version we're using in AWS-land? Or maybe it doesn't matter to keep these in lockstep?

Comment (mjgiarlo) [reply Dockerfile-es]: Dang, just a second too late! :laughing:

Comment (justinlittman) [reply Dockerfile-es]: We are using 6.4 in AWS-land.

Comment (mjgiarlo) [reply Dockerfile-es]: :tada: thanks, @justinlittman

**Pull request merged**